test/unit/org/apache/cassandra/cql3/DeleteTest.java

```diff
@@ -93,6 +93,33 @@ public void prepare() throws Exception
         pstmt5 = session.prepare("select id, cid, inh_c, val from junit.tpc_inherit_c where id=? and cid=?");
     }
 
+    @Test
+    public void deleteCasWithStatic()
```

Comment:
ill pull this out into its own ticket... this should be fixed in mainline branches and not accord